bmc/engine/check_settling_time.h

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
Is this header following the standard of DSVerifier? If we have no standard yet, may I ask you to propose one?